decidim-core/app/functions/decidim/core/participatory_space_finder_base.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    # A very basic resolver for the GraphQL endpoint for a single participatory spaces
+    # This can be easily overwritten by the participatory_space_manifest.query_finder
```

Comment:
```suggestion
    # This is the defult resolver if none is specified. To use a custom one define the 
   +participatory_space_manifest.query_finder+ property.
```

Reply:
I've updated all the comment for these files. Hopefully it will be clearer the meaning of each one of them now.